cluster_agent/jobbergate/submit.py

```diff
@@ -86,10 +83,10 @@ async def submit_job_script(
             or SETTINGS.DEFAULT_SLURM_WORK_DIR
         )
 
-        local_script_path = submit_dir / f"{name}.job"
-        local_script_path.write_text(job_script)
-
-    logger.debug(f"Copied job_script to local file {local_script_path}.")
+        for path, file_content in pending_job_submission.job_script_files.files.items():
+            local_script_path = submit_dir / path
+            local_script_path.write_text(file_content)
```

Comment:
One bug I solved after testing, we need to create some directories if necessary, because we are keeping the path provided by the application sript.

Reply:
solved in 9afc415bffa3648cfba30d1413fe806799caff70